### PR TITLE
Reference other relevant open source projects

### PR DESCRIPTION
**Pull request by imatiach-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -310,12 +310,26 @@ Issue](https://help.github.com/articles/creating-an-issue/).
 
 * [Microsoft Cognitive Toolkit](https://github.com/Microsoft/CNTK)
 
+* [LightGBM](https://github.com/Microsoft/LightGBM)
+
+* [Recommenders](https://github.com/Microsoft/Recommenders)
+
+* [ML.NET](https://github.com/dotnet/machinelearning)
+
+* [Infer.NET](https://github.com/dotnet/infer)
```
Comment (mhamilton723) [inline README.md]: Lets remove until we have an integration story with them

File: README.md [hunk 2]
```diff
@@ -310,12 +310,26 @@ Issue](https://help.github.com/articles/creating-an-issue/).
...
+
+* [DMTK: Microsoft Distributed Machine Learning Toolkit](https://github.com/Microsoft/DMTK)
+
+* [LightLDA](https://github.com/Microsoft/LightLDA)
```
Comment (mhamilton723) [inline README.md]: Likewise

File: README.md [hunk 3]
```diff
@@ -310,12 +310,26 @@ Issue](https://help.github.com/articles/creating-an-issue/).
...
+
 * [Azure Machine Learning
```
Comment (mhamilton723) [inline README.md]: This link needs to be changed to something new

**Comment by imatiach-msft** [comment]
Kept these (note LightGBM is part of DMTK, and we should keep recommenders since that is where SAR is from):
* [LightGBM](https://github.com/Microsoft/LightGBM)
* [Recommenders](https://github.com/Microsoft/Recommenders)
* [DMTK: Microsoft Distributed Machine Learning Toolkit](https://github.com/Microsoft/DMTK)

File: README.md (hunk 1 above)
Comment (imatiach-msft) [reply README.md]: done!

File: README.md (hunk 2 above)
Comment (imatiach-msft) [reply README.md]: done!

File: README.md (hunk 3 above)
Comment (imatiach-msft) [reply README.md]: the azure machine learning service?  Not sure which link it should reference.  Keeping this unmodified.

**Pull request merged**